.changeset/twenty-cameras-take.md

```diff
@@ -5,6 +5,14 @@
 "section": other
 ---
 
-Untangle `PresenceStates` methods and properties
+Presence package updates
 
-`PresenceStatesEntries` which represent each of the states in `PresenceStates` schema have been relocated from directly within `PresenceStates` to under property names `prop`. Only the `add` method remains. (Type `PresenceStatesMethods` has been removed.)
+### The methods and properties of `PresenceStates` have been reorganized ([#23031](https://github.com/microsoft/FluidFramework/pull/23031))
```

Comment:
This will end up rendered as:

Title (#23031) (#23021)

The originating PR is always appended.

Reply:
Will it be added here or on line 8 (the title)?
In prior discussion you mentioned adding PR references directly in changesets when changelog is added after. Not sure to get the right thing to happen.